source/install/cloud-changelog.md

```diff
@@ -3,12 +3,58 @@
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
 Latest Mattermost Cloud releases:
+
+- [Release 2022-02-16](#release-2022-02-16)
 - [Release 2022-02-10](#release-2022-02-10)
 - [Release 2022-02-03](#release-2022-02-03)
 - [Release 2022-01-27](#release-2022-01-27)
 - [Release 2022-01-25](#release-2022-01-25)
 - [Release 2022-01-11](#release-2022-01-11)
-- [Release 2021-12-08](#release-2021-12-08)
+
+## Release 2022-02-16
+
+### Improvements
+
+#### User Interface (UI)
+ - Users are now able to use the channel switcher to search channels across teams if the users are part of more than one team.
+ - Updated the recommended minimum supported Firefox version to v91+.
+ - Clarified in-product error string "Oops!" as "Unable to continue" for both translators and target audiences in cases where a user doesn't have sufficient permissions to add users or guests.
+ - Removed incorrect in-product string text from the **Send full message contents** email notification option description displayed via **System Console > Site Configuration > Notifications**.
+ - Removed incorrect in-product string text from the **Send full message contents** email notification option description displayed via **System Console > Site Configuration > Notifications**. 
+ - Added the ability to send unsanitized user to the source user on ``user_updated`` event.
+ - In the compact view, the sender's username is now always shown on posts.
+ - Added a "Create board" button to the Channel Intro section.
```

Comment:
```suggestion
 - Added a **Create board** button to the Channel Intro section.
```

Reply:
I'm not clear on what's we mean by Channel Intro section.

Reply:
Some screenshots here: https://github.com/mattermost/mattermost-webapp/pull/9683. When a new channel is created, there are some buttons and a channel intro text.

Reply:
Thank you! I wonder if we have an elegant way to refer to that part of the UI. @justinegeffen - do we have a standard way to refer to the UI that's visible when someone joins a brand new channel?

Reply:
> Thank you! I wonder if we have an elegant way to refer to that part of the UI. @justinegeffen - do we have a standard way to refer to the UI that's visible when someone joins a brand new channel?

Great question. I'd say "Channel landing page" except it's not really a page. Perhaps "Channel intro header" would work? Looking at the screenshot in the linked ticket I'd say that might work better?